### DISCUSSION TITLE
Support for WebAssembly Garbage Collector (Wasm GC)

### DISCUSSION THREAD

**Discussion by GomezMig03** [Floorp-Projects/Floorp#1274] [Feature Request]
Hi,
I noticed that Floorp currently doesn't support the WebAssembly Garbage Collector (Wasm GC) feature, as referenced in the [Feature Extensions - WebAssembly]( https://webassembly.org/features/).
![image](https://github.com/user-attachments/assets/804a7498-a403-4d8f-8667-258448137764)
I was wondering if there was any plans on adding this feature on the future.

For context, Wasm GC is used in applications like Google Sheets, where it has been shown to significantly improve performance, as detailed in this [Google Sheets Case Study](https://web.dev/case-studies/google-sheets-wasmgc).

**Comment by CutterKnife** [comment]
Floorp is based on Firefox ESR. It will be supported by the end of Firefox ESR 115 support at the latest.
